third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -279,8 +279,14 @@ func testAccComposerEnvironmentDestroy(s *terraform.State) error {
 func testAccComposerEnvironment_basic(name string) string {
 	return fmt.Sprintf(`
 resource "google_composer_environment" "test" {
-  name           = "%s"
-  region         = "us-central1"
+	name           = "%s"
+	region         = "us-central1"
+
+	config {
```

Comment:
Does the test still work without this block? My philosophy is that the basic test should be, for the most part, the minimum that you need in order to get a resource provisioned.

Reply:
Thanks, looks like I didn't need to add this block. Updated.